spec/lib/group_creation_spec.rb

```diff
@@ -1,7 +1,7 @@
 # spec/lib/group_creation_spec.rb
 require 'spec_helper'
 
-describe GroupCreation do
+describe Group::GroupCreator do
```

Comment:
we should move all this specs under their own engine, another thing is making them green of course. Next PR.